pages/pipelines/controlling_concurrency.md

```diff
@@ -47,7 +47,7 @@ Concurrency groups guarantee that jobs will be run in the order that they were c
 ## Concurrency and parallelism
 
 Sometimes you need strict concurrency while also having jobs that would benefit from parallelism.
-In these situations you can use *concurrency gates* to control which jobs run in parallel and which jobs run one at a time.
+In these situations you can use *concurrency gates* to control which jobs run in parallel and which jobs run one at a time. Concurrency gates come in pairs, for it to work, when you open a concurrency gate, you have to close it.
```

Comment:
I think the sentence is a bit too complex. How about removing one of the comma sections (and add a missing comma elsewhere):

```suggestion
In these situations, you can use *concurrency gates* to control which jobs run in parallel and which jobs run one at a time. Concurrency gates come in pairs, so when you open a gate, you have to close it.
```

What do you think? We could also split it into multiple sentences.

Reply:
@mbelton-buildkite Yes we definitely can